config.py

```diff
@@ -97,7 +97,8 @@ class Config(object):
     RAS_NOTIFY_ACCOUNT_LOCKED_TEMPLATE = os.getenv('RAS_NOTIFY_ACCOUNT_LOCKED_TEMPLATE', 'account_locked_id')
     SEND_EMAIL_TO_GOV_NOTIFY = os.getenv('SEND_EMAIL_TO_GOV_NOTIFY', False)
     REQUESTS_POST_TIMEOUT = os.getenv('REQUESTS_POST_TIMEOUT', 20)
-    WTF_CSRF_ENABLED = bool(strtobool(os.getenv('WTF_CSRF_ENABLED', "True")))
+    SECURE_APP = bool(strtobool(os.getenv('SECURE_APP', "True")))
```

Comment:
We probably want SECURE_APP set to False in the Development and Testing Config objects